src/logging/monitor.ts

```diff
@@ -55,11 +54,26 @@ async function* splitStream(chunkIterable: AsyncIterable<any>) {
  * Streams logs from journalctl and calls container hooks when a record is received matching container id
  */
 class LogMonitor {
+	// Additional host services we want to stream the logs for
+	private HOST_SERVICES = [
```

Comment:
Not pushing too much for this, but I think this would be better as a constant in the module. That would avoid the need for using `this.` to reference it